apps/portal/controllers/apis/login.jag

```diff
@@ -96,13 +102,13 @@ var user;
             }
             else {
                 if (dest) {
-                    response.sendRedirect(dest + "login?loginAttempt=true");
+                    response.sendRedirect('/portal/login?loginAttempt=true');
                 }
             }
         }
         else {
             if (dest) {
```

Comment:
If we do not have a dest, then this is direct API call coming outside from the portal. For instance, in the ES integration, ES calls this API. So in that case, we do not need to redirect to login UI page. Please do correct me, this this is not the case.